presto-main/src/test/java/com/facebook/presto/operator/TestHashJoinOperator.java

```diff
@@ -116,7 +116,7 @@ public void testInnerJoin(boolean parallelBuild, boolean probeHashEnabled, boole
         );
 
         // expected
-        MaterializedResult expected = MaterializedResult.resultBuilder(taskContext.getSession(), concat(probePages.getTypes(), buildPages.getTypes()))
+        MaterializedResult expected = MaterializedResult.resultBuilder(taskContext.getSession(), concat(probePages.getTypesWithoutHash(), buildPages.getTypesWithoutHash()))
```

Comment:
But why? I wanted the changes to be atomic. Also I think getTypesWithoutHash method addition is worth mentioning in the commit log for the ones reading it :)